pallets/hyperdrive/src/tezos.rs

```diff
@@ -229,11 +233,24 @@ fn registration_payload_schema() -> &'static Micheline {
     })
 }
 
-/// Parses an encoded [`RawAction::RegisterJob`] action's payload into [`JobRegistration`].
-///
-/// # Example
-/// A message's payload to register a job could look like:
+/// The structure of a [`RawAction::DeregisterJob`] action before flattening:
 ///
+/// ```txt
+/// sp.TRecord(
+///     jobId=sp.TNat,
+/// ).right_comb()
+/// ```
+#[cfg_attr(rustfmt, rustfmt::skip)]
+fn registration_job_id_schema() -> &'static Micheline {
```

Comment:
It is specific to `deregister_job` action, right?
```suggestion
fn deregistration_job_schema() -> &'static Micheline {
```